benchmarks/json_data.py

```diff
@@ -0,0 +1,43 @@
+import json
```

Comment:
Rename this Python file to `generate_data.py`.

Reply:
Done